### PR TITLE
Utilities/StaticAnalyzers: EventSetupRecord get report without modules in IB filtering.

### PR DESCRIPTION
**Pull request by gartung**
Removed the filtering in the EventSetupRecord::get report for modules in IB validation. This increases the size of event-setup-record-get.yaml to 412K as all calls are reported.

### CODE REVIEW

**Comment by makortel** [comment]
@cmsbuild, please tesst

**Comment by makortel** [comment]
> This increases the size of event-setup-record-get.yaml to 412K as all calls are reported.

@smuzaffar Is 412 kiB report per IB still acceptable?

**Comment by smuzaffar** [comment]
> @smuzaffar Is 412 kiB report per IB still acceptable?

this is fine @makortel

File: Utilities/StaticAnalyzers/scripts/callgraph.py [hunk 1]
```diff
@@ -57,17 +57,17 @@
...
+#with open('modules_in_ib.txt', 'r') as m:
+#    for line in m:
+#        modules.append(line.strip())
+#mods='|'.join(modules)
+#mods_regex=r'\b(' + mods+ r'QQQQQQQQ)\b'
+#comp_mods_regex=re.compile(mods_regex)
+#n=comp_mods_regex.search(function)
+#assert(n)
+#n2=comp_mods_regex.search(function2)
+#assert(n2)
```
Comment (makortel) [inline Utilities/StaticAnalyzers/scripts/callgraph.py:70]: Is it useful to keep these in comments (wrt. digging out from git history)?

**Comment by makortel** [comment]
@gartung Could also remove the `modules_in_ib.txt` now that it is not needed anymore?

File: Utilities/StaticAnalyzers/scripts/callgraph.py (hunk 1 above)
Comment (gartung) [reply Utilities/StaticAnalyzers/scripts/callgraph.py]: No

**Comment by makortel** [comment]
@cmsbuild, please test

**Comment by makortel** [comment]
+1

**Comment by perrotta** [comment]
+1

**Pull request merged**